video/out/gpu/video.c

```diff
@@ -3849,6 +3849,9 @@ static void gl_video_dr_free_buffer(void *opaque, uint8_t *data)
 struct mp_image *gl_video_get_image(struct gl_video *p, int imgfmt, int w, int h,
                                     int stride_align)
 {
+    if (!gl_video_check_format(p, imgfmt))
```

Comment:
commit message:

* DR == direct rendering? Sorry, I guess this is pretty standard seeing as how it's in at least the name of the function before this one.
* Previous behaviour should be past tense.
* Sp -> So? Also this isnt really a sentence
* characteristcs **are** up to the driver